lib/jekyll/frontmatter_defaults.rb

```diff
@@ -130,7 +130,7 @@ def valid?(set)
     # new_scope - the new scope hash
     #
     # Returns true if the new scope has precedence over the older
-    def has_precedence?(old_scope, new_scope)
+    def precedence?(old_scope, new_scope)
```

Comment:
Ok, I see that this convention is already implemented in most other parts of Jekyll.